.gitignore

```diff
@@ -1,4 +1,5 @@
 .*
 !.gitignore
+!.travis*
 *.deb
 opt/cs50/bin/phpliteadmin
```

Comment:
Should we move this to top-level now, as we've done for other apps, since it's just one file?